source/2019-05-20-ember-2019-roadmap-call-for-posts.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Ember's 2019 Roadmap: Call for Blog Posts"
+author: Katie Gengler
+tags: Recent Posts, Roadmap, 2019
+responsive: true
+---
+
+Ember in 2018 (and 2019) has been more exciting than ever! We've been working diligently to advance the project based on our [2018 Roadmap](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md), but while we work on that, it is time to think about the future. 
+
+The Ember team would like you to write a blog post to propose goals and direction for Ember for the next year. The content of these posts will help us to draft our next Roadmap RFC.
+
+# What's a Roadmap RFC?
+
+The Roadmap will outline the community's priorities for the next year. It will be presented as an RFC and follow [the normal RFC process](https://github.com/emberjs/rfcs). The RFC will be written based on community input--the blog posts we've requested and other community discussion--and curation from the core teams.
+
+This is will be our second annual Roadmap RFC. To see the results of the process, see [the 2018 Roadmap RFC](https://github.com/emberjs/rfcs/blob/master/text/0364-roadmap-2018.md). 
+
+The 2018 Roadmap led to the ["Octane Edition" of Ember, currently in preview](https://emberjs.com/editions/octane/). Octane [includes many features](https://github.com/emberjs/ember.js/issues/17234), most of which have already landed in a stable release. As the edition nears completion, we want the community to think beyond Octane to the next steps for Ember. The resulting Roadmap for 2019 will not necessarily include an edition but will outline what we will be working toward. 
+
+**The Roadmap process is part of our on-going effort to have shared, clear, and published project-wide goals.**
+
+# #EmberJS2019
+
+To contribute a post: Tweet a link to the post with the hashtag #EmberJS2019 or email a link to roadmap@emberjs.com. These posts will be collected and categorized, and each one will be read by those working to draft the Roadmap RFC.
+
+### _Format_
```

Comment:
this should be an H2 if the one before it is an H1

Reply:
/shrug I used same as last year

Reply:
I'm sorry I didn't catch it then- it's not good for accessibility.